packages/edit-site/src/components/page-templates-template-parts/index.js

```diff
@@ -248,35 +266,36 @@ export default function DataviewsTemplates() {
 				maxWidth: 400,
 				minWidth: 320,
 				enableSorting: false,
+			} );
+		}
+		// TODO: This should be improved when we implement field ordering with an API.
```

Comment:
What does this mean? Can you expand on this?

Reply:
The plan is to support fields reordering, which would require an API like `order` or something. Right now I'm creating  the array -> adding a conditional field in the wanted order (3rd) -> add another one after that. With ordering API I believe we wouldn't need to construct the fields array like that.